src/main/java/com/faforever/client/news/NewsController.java

```diff
@@ -108,38 +78,15 @@ public void onUnreadNewsEvent(UnreadNewsEvent unreadNewsEvent) {
 
 
   private void loadNews() {
-    newsService.fetchNews().thenAccept(newsItems -> {
-      Platform.runLater(() -> {
-        newsListView.getItems().setAll(newsItems);
-        onLoadingStop();
-        if (!newsItems.isEmpty()) {
-          NewsItem mostRecentItem = newsItems.get(0);
-          preferencesService.getPreferences().getNews().setLastReadNewsUrl(mostRecentItem.getLink());
-          preferencesService.storeInBackground();
-        }
-        newsListView.getSelectionModel().selectFirst();
-      });
+    Platform.runLater(() -> {
+      Website website = clientProperties.getWebsite();
+      newsWebView.getEngine().load(website.getNewsHubUrl());
+      onLoadingStop();
```

Comment:
The loading should be removed the way it is now it is technical impossible to see the loading spinner

Reply:
Well or listen on the engine finishing the loading